yolox/utils/logger.py

```diff
@@ -438,3 +438,329 @@ def initialize_wandb_logger(cls, args, exp, val_dataset):
                     wandb_params.update({k[len(prefix):]: v})
 
         return cls(config=vars(exp), val_dataset=val_dataset, **wandb_params)
+
+import importlib.metadata
```

Comment:
The logger file might be too large, I highly recommend to use a new file named `mlflow_logger.py`.

Reply:
created a new file and shifted the code there